core/src/databases/table_schema.rs

```diff
@@ -259,6 +275,10 @@ impl TableSchema {
                         }
                     }
                     Some(Value::String(s)) => Ok(SqlParam::Text(s.clone())),
+                    Some(Value::Object(obj)) => match Self::try_parse_date_object(obj) {
+                        Some(date) => Ok(SqlParam::Text(date)),
```

Comment:
Actually hesitating here. Should I actually make a `DateTime` SqlParam and have its rusqlite-specific ToSql do this conversion ?
That may be more future-proof if we start having several ToSql implementations for different DBs